test/gtest/uct/test_ib.cc

```diff
@@ -481,13 +481,13 @@ UCS_TEST_P(test_uct_wakeup_ib, txrx_cq)
     /* make sure the file descriptor is signaled */
     ASSERT_EQ(poll(&wakeup_fd, 1, 1), 1);
 
-    status = uct_wakeup_wait(wakeup_handle);
```

Comment:
Because old logic of the `wait()` was `clear_fd` and `wait on fd`. So, it hangs with old logic. In case of the test we need just acknowledgments for the events.